### DISCUSSION TITLE
Voltage-clamp stimuli

### DISCUSSION THREAD

**Discussion by RichardBJ** [BlueBrain/BluePyOpt#502] [Q&A]
Hi we have been looking at the API and wondered if there was a mechanism to optimise to voltage clamp stimuli, rather than current clamp? (bluepyopt.ephys.stimuli.LFPStimulus).  We are investigating coding this separately, but don't want to re-invent the wheel!
Kind Regards
Richard

**Comment by AurelienJaquier** [comment]
Hi @RichardBJ ! Sorry, we don't have any voltage clamp stimuli in BluePyOpt.

**Reply by RichardBJ** [reply]
Thanks for the speedy reply!!! It's something we'll look into then

**Reply by AurelienJaquier** [reply]
Alright! Don't hesitate to reach if you have any question
